deploy/openshift/tools/docker-config-generate/generate-docker-config.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+get_parameters() {
+    read -p "Docker registry: "  REGISTRY
+    read -p "Docker username: "  USERNAME
+    read -s -p "Docker secret: "  SECRET
+}
+
+generate_base64() {
+    AUTH=$(echo $USERNAME:$SECRET | base64 -w 0)
+    RENDERED_JSON=$(echo {\"auths\":{\"${REGISTRY}\":{\"username\": \"${USERNAME}\",\"password\":\"${SECRET}\",\"auth\":\"${AUTH}\"}}})
+    DOCKER_CONFIG=$(echo $RENDERED_JSON | base64 -w 0)
+}
```

Comment:
Unfortunately base64 on macos does not have the -w option. And most of the people that are going to be using this, are using macos. I've seen suggestions to use base64 encoder built in openssl (e.g. `openssl base64`) that works fine on most OS's and doesn't add new lines.

Reply:
So I would try changing `base64 -w 0` to `openssl base64` and adding a comment about that it is used for compatibility reasons.

Reply:
Done